packages/types-cloudflare-worker/package.json

```diff
@@ -28,5 +28,8 @@
     "global.js",
     "global.d.ts",
     "LICENSE"
-  ]
+  ],
+  "devDependencies": {
+    "@types/node": "^12.0.8"
```

Comment:
Any reason for not using latest?

Reply:
Because it's been a month. :wink: I'll update.